### PR TITLE
prevent spam when setting size to -1

### PR DESCRIPTION
**Pull request by jtnord**
the size for every plugin is set to `-1` which creates excessive spam when running.

prevents spam like the following
```
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for SBuild updating size from 19,704 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for SSSCM updating size from 11,287 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for StashBranchParameter updating size from 1,206,570 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for Surround-SCM-plugin updating size from 41,279 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for TestComplete updating size from 105,057 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for TestFairy updating size from 2,277,975 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for TwilioNotifier updating size from 2,146,774 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for URLSCM updating size from 16,977 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for abap-ci updating size from 347,171 to -1
Apr 12, 2024 11:02:22 AM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for absint-a3 updating size from 36,542 to -1
```

log these at FINE level, log all other updates and INFO level

### Testing done

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/update_center/MockUpdateCenter.java [hunk 1]
```diff
@@ -227,7 +228,11 @@ private void updating(JSONObject plugin, String key, Object val) throws JSONExce
         Object old = plugin.opt(key);
         plugin.put(key, val);
         if (!String.valueOf(val).equals(String.valueOf(old))) {
-            LOGGER.log(Level.INFO, "for {0} updating {1} from {2} to {3}", new Object[] {plugin.getString("name"), key, old, val});
+            if (MINUS_ONE == val && "size".equals(key)) {
```
[the remark this answers is not in the document]
Comment (jtnord) [reply src/main/java/org/jenkinsci/test/acceptance/update_center/MockUpdateCenter.java:231]: eipic failure -> https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-1529/1/testReport/plugins/AntPluginTest/latest_linux_jdk21_firefox_split8___testWithAntPipelineBlock/

```
Apr 12, 2024 1:20:09 PM org.jenkinsci.test.acceptance.update_center.MockUpdateCenter updating
INFO: for virtualbox updating size from 6,592,022 to -1
```

[the remark this answers is not in the document]
Comment (jtnord) [reply src/main/java/org/jenkinsci/test/acceptance/update_center/MockUpdateCenter.java]: we where using `"-1" ` not `-1` so it was a `String`

**Comment by jtnord** [comment]
https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-1529/3/realtimeTestReport-512/plugins/AntPluginTest/latest_linux_jdk21_firefox_split2___testWithAntPipelineBlock/ looking much "better" :)

**Comment by timja** [comment]
https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-1529/3/testReport/plugins/AntPluginTest/latest_linux_jdk21_firefox_split2___testWithAntPipelineBlock/

(realtime test report just redirects to the build page after its finished I guess)

**Review by timja — APPROVED** [review]

**Pull request merged**